pkg/cfn/builder/api_test.go

```diff
@@ -2147,6 +2146,31 @@ var _ = Describe("CloudFormation template builder API", func() {
 		})
 	})
 
+	Context("with Fargate profiles", func() {
```

Comment:
@marccarre unit test added, I checked for all resouces as well length of array as well.